### PR TITLE
Update to 4.11

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^9.5"
     },
     "config": {
         "process-timeout": 600,
         "platform": {
-            "php": "7.1.0"
+            "php": "7.4.0"
```
Comment (GuySartorelli) [inline composer.json:16]:
```suggestion
            "php": "7.4.0"
        },
        "allow-plugins": {
            "composer/installers": true,
            "silverstripe/recipe-plugin": true,
            "silverstripe/vendor-plugin": true
        }
```

File: composer.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
-        "silverstripe/dynamodb": "^4.0",
-        "silverstripe/crontask": "*@dev"
+        "php": ">=7.4.0",
+        "silverstripe/installer": "^4.11",
+        "silverstripe/dynamodb": "^4.1",
+        "silverstripe/crontask": "^2.5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^9.5"
```
Comment (GuySartorelli) [inline composer.json:11]: I'd be inclined to just remove this - the demo site has no tests.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json:11]: Removed

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:16]: Added

File: composer.json [hunk 3]
```diff
@@ -2,49 +2,46 @@
...
+        "silverstripe/crontask": "^2.5"
     },
     "config": {
         "process-timeout": 600,
         "platform": {
-            "php": "7.1.0"
+            "php": "7.4.0"
         }
     },
+    "allow-plugins": {
```
Comment (GuySartorelli) [inline composer.json:16]: This has to be inside the config
```json
   "config": {
        "process-timeout": 600,
        "platform": {
            "php": "7.4.0"
        },
        "allow-plugins": {
            "composer/installers": true,
            "silverstripe/vendor-plugin": true,
            "silverstripe/recipe-plugin": true
        }
    },
```

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**